MOO market | Add MOOConfiguratorCelo and MOOConfiguratorAlfajores

x-moola opened the pull request:
For [AMM-1370 [Solidity] Create a MOO configurator contract](https://moolamarket.atlassian.net/browse/AMM-1370)

```
LTV = 50% 
Liquidation Threshold = 65% 
Liquidation Bonus = 10% 
Revenue Factor = 10% 
```

- Add configurator contracts for adding MOO reserve on Alfajores and Mainnet
- Update other related files to go with the new reserve

lastperson commented on contracts/protocol/lendingpool/MOO/MOOConfiguratorAlfajores.sol:
```diff
@@ -0,0 +1,93 @@
+  address constant aTokenImpl = 0xe8B286649713447D8d5fBeBC28c731830d19B6C9;
+  address constant stableDebtTokenImpl = 0xB6a5059A228a16Fa2827E28E52ceC96BBC63D639;
+  address constant variableDebtTokenImpl = 0xB65b6a6a6F78E4daABF259c756567ae346699687;
+  address constant interestRateStrategyAddress = 0x5B41b0c78659636c6664f08F7cCb620ceA3F1206;
```
I guess interest rate strategy should be the same as CELO, because it is much closer in volatility to MOO.

lastperson reviewed (changes requested):
Interest rate strategy need to be changed.

x-moola commented:
Updated to use `rateStrategyWETH` in both configurator files and updated `markets/moola/reservesConfigs.ts`
https://github.com/moolamarket/moola-v2/pull/20/commits/a0ea2897072fd3ef060c8b0434bcb86f95ede243
